.github/workflows/tests.yaml

```diff
@@ -44,13 +44,14 @@ jobs:
         python-version: ${{ matrix.pyv }}
     - name: install
       run: |
-        pip install ".[all,tests]"
+        pip install wheel
+        pip install -e ".[all,tests]"
```

Comment:
Coverage was being counted from both local directory and the installed package before, thus reducing the coverage. Also, it's a good practice to use local directory for running the tests (hence, the _editable_ mode).

`pip install wheel`, because I saw a lot of packages were being installed on legacy mode.